includes/assets/stories.css

```diff
@@ -0,0 +1,142 @@
+/*
```

Comment:
Is this css file automatically minified?

Reply:
We don't have the infrastructure for this yet, sadly. Something we'll need to work on with a growing set of CSS files. But that's independent of this work